code/client/src/pages/List.jsx

```diff
@@ -126,8 +128,54 @@ const List = () => {
       w.address !== ONEConstants.EmptyAddress
   }
 
+  const getBase64 = (data) => new Promise((resolve) => {
+    const reader = new FileReader()
+    reader.addEventListener('load', () => resolve(reader.result))
+    reader.readAsText(data)
+  })
+
+  const handleImport = async (info) => {
```

Comment:
probably two files will need to be imported - the layers (partial proofs), and the state (in redux). You can also combine them in one binary blob during export, so import can be done using only one file. To do that, you might need protobuf and define a new .proto.

Reply:
Not very sure about the layers thing... I need to check it out.

Reply:
good reference is in `Create.jsx`  - check out what is stored